dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -239,8 +239,11 @@ private boolean problemFilter(LinkageProblem linkageProblem) {
     String targetClassName = symbol.getClassBinaryName();
     String methodName = symbol.getName();
 
-    // Skip references to Java runtime class. For example, java.lang.String.
-    if (classDumper.isSystemClass(targetClassName)) {
+    if (ClassDumper.isArrayClass(targetClassName)) {
+      // Skip references to array class. However, we want to check other JDK-provided classes,
```

Comment:
delete this comment. It describes what the code is not doing.

Reply:
Removed.